docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions.md

```diff
@@ -249,3 +286,82 @@ To refresh group memberships,
 1. Click **Refresh Group Memberships**.
 
 **Result:** Any changes to the group members' permissions will take effect.
+
+## Restricted Admin
+
+:::warning Deprecated
+
+The Restricted Admin role is deprecated, and will be removed in a future version of rancher (2.10 or higher). Users are encouraged to make a custom role with the desired permissions rather than relying on this built-in role).
+
+:::
+
+A new `restricted-admin` role was created in Rancher v2.5 in order to prevent privilege escalation from the local Rancher server Kubernetes cluster. This role has full administrator access to all downstream clusters managed by Rancher, but it does not have permission to alter the local Kubernetes cluster.
```

Comment:
```suggestion
Rancher v2.5 introduced the `restricted-admin` role, to prevent privilege escalation originating from the upstream (local) Rancher cluster. This role has full administrator access to all downstream clusters managed by Rancher, but it doesn't have permission to alter the local Kubernetes cluster.
```

Reply:
This comment is not correct - the escalation would be targeting the local cluster, not originating from the local cluster. I'll try to integrate new language into the sentence accordingly.